### ISSUE TITLE
what if we want to use deepstream-app -c <config>

### ISSUE THREAD

**Issue by amiroft** [mrscarletzhou/retinaface-deepstream-python#6]
Thanks for your nice effort
what if we want to use deepstream-app -c <config>?
what steps should we take ?
is it possible at all?

**Comment by zhouyuchong** [comment]
@amiroft depends what you want to achieve. If you just want to simply see the output display, just replace the configs of PGIE. If you want to take a step further adding your own logical codes,  you should modify the original deepstream-app c codes by adding probes.

**Comment by amiroft** [comment]
Thanks for your reply
at first stage just seeing that output display is enough
I want to deploy this pipeline on jetson nano is it still possible with model repo you mentioned?
since every time I try (even with other models I face error(probably related to model ) I tried onnx or .engine ones 
is there any way to find the solution for this part?

**Comment by zhouyuchong** [comment]
@amiroft Yes you can use it on JetsonNano, just be careful about *nvbuf-memory-type* property. 

For Q2, you can test your *onnx* or *tensorrt* files first then apply them to deepstream pipeline.
